numba/tests/test_builtins.py

```diff
@@ -838,6 +838,27 @@ def test_min_1_invalid_types_npm(self):
         with self.assertTypingError():
             self.check_min_max_invalid_types(min_usecase1, flags=no_pyobj_flags)
 
+    def check_minmax_bool1(self, pyfunc, flags):
+        cr = compile_isolated(pyfunc, (types.bool_, types.bool_), flags=flags)
+        cfunc = cr.entry_point
+
+        x_operands = [False, True]
+        y_operands = [False, True]
+        for x, y in itertools.product(x_operands, y_operands):
```

Comment:
```suggestion
        operands = (False, True)
        for x, y in itertools.product(operands, operands):
```
`x_operands` and `y_operands` are the same, only need one definition?